compiler/p/codegen/PPCTableOfConstants.cpp

```diff
@@ -430,6 +430,9 @@ int32_t TR_PPCTableOfConstants::lookUp(double dvalue, TR::CodeGenerator *cg)
          return PTOC_FULL_INDEX;
       }
 
+   if (comp->getOption(TR_DisableTOC))
+      return PTOC_FULL_INDEX;
+
```

Comment:
Let's put this as the first check, it's cheaper than the one above it and it should overrule the others.

Reply:
2163152ff